tests/ops/gaussian.py

```diff
@@ -0,0 +1,42 @@
+import torch
+
+import pyro.distributions as dist
+from pyro.ops.gaussian import Gaussian
+from tests.common import assert_close
+
+
+def random_gaussian(batch_shape, dim, rank=None):
```

Comment:
This test helper is moved from tests/ops/test_gaussian.py. I've changed rank logic so that the default is to use high-rank components, thereby improving the conditioning of `precision`.